groceries_app/views.py

```diff
@@ -10,5 +11,10 @@ class AboutView(TemplateView):
 class RecipesView(TemplateView):
     template_name = 'recipes/recipes.html'
 
-class PlannerView(TemplateView):
-    template_name = 'grocery_planner/planner.html'
\ No newline at end of file
+class MealChoiceView(TemplateView):
+    template_name = 'grocery_planner/gp1-meal-choice.html'
+    
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
```

Comment:
What does `context` mean here?

Reply:
https://stackoverflow.com/questions/20957388/what-is-a-context-in-django